### PR TITLE
Add HTTPHeaderFields to RequestType

### PR DESCRIPTION
**Pull request by ishkawa**

### CODE REVIEW

File: APIKit/RequestType.swift [hunk 1]
```diff
@@ -49,6 +53,10 @@ public protocol RequestType {
 
 /// Default implementation of RequestType protocol
 public extension RequestType {
+    public var HTTPHeaderFields: [String: String] {
```
Comment (ikesyo) [inline APIKit/RequestType.swift]: This should be placed under `public var parameters`.

File: APIKit/RequestType.swift [hunk 2]
```diff
@@ -100,6 +109,10 @@ public extension RequestType {
         URLRequest.HTTPMethod = method.rawValue
         URLRequest.setValue(requestBodyBuilder.contentTypeHeader, forHTTPHeaderField: "Content-Type")
         URLRequest.setValue(responseBodyParser.acceptHeader, forHTTPHeaderField: "Accept")
+        
+        for (key, value) in HTTPHeaderFields {
```
Comment (ikesyo) [inline APIKit/RequestType.swift]: We can use `forEach` here.

**Comment by ikesyo** [comment]
It's a nice addition! Just a few nitpicks. :smiley:

**Comment by ishkawa** [comment]
Sorry for my late response. I've fixed them :sparkles:

**Comment by ikesyo** [comment]
:+1:

**Pull request merged**